core/classifiers/CodeClassifier/CodeClassifier.py

```diff
@@ -0,0 +1,481 @@
+# coding: utf-8
+#
+# Copyright 2017 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Base class for classification algorithms"""
+
+from collections import Counter
+import keyword
+from StringIO import StringIO
+import token
+import tokenize
+
+from core.classifiers.BaseClassifier import BaseClassifier
+
+import numpy as np
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.model_selection import GridSearchCV
+from sklearn.svm import SVC
+
+# pylint: disable=invalid-name
+def _get_tokens(program):
+    """Generate tokens for program using tokenize module."""
+    for tid, tname, _, _, _ in tokenize.generate_tokens(
+            StringIO(program).readline):
+        yield (tid, tname)
+
+
+def _cv_tokenizer(program):
+    """Tokenize Python program for CountVectorizer."""
+    token_program = []
+    for tid, tname in _get_tokens(program):
+        if tid == token.N_TOKENS or tid == 54:
+            continue
+        elif tid == token.NAME:
+            if tname in keyword.kwlist:
+                token_program.append(tname)
+            else:
+                token_program.append('V')
```

Comment:
I think you should explain that you are using 'V' for variables here. Also a good idea to assign this to a variable.